src/components/AlreadyElectrifiedChart.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+const electrifiedColor = "rgb(163, 215, 164)"
+const fossilColor = "rgb(255, 87, 34)"
+const AlreadyElectrifiedChart = ({ label, electrifiedPct, fossilPct }) => { //otherPct
+  const formattedData = {
+
+  }
+  if (electrifiedPct !== 0 || fossilPct !== 0) {
+    return (
+      <svg width={"100%"} height="50px" viewBox="0 0 300 15" xmlns="http://www.w3.org/2000/svg" style={{ margin: '.5em 0 0 0' }}>
```

Comment:
Nice use of a quick SVG! We should have some kind of alt text to communicate this information to screen readers, especially since it's basically just a percentage value.

Bootstrap has the [`.sr-only`](https://getbootstrap.com/docs/4.6/utilities/screen-readers/) class which should work to create an invisible text alternative to this SVG. I also think adding the `<title>` attribute to the SVG would work. You can learn more from [CSS Tricks' guide to accessible SVGs](https://css-tricks.com/accessible-svgs/).

Reply:
Thanks @vkoves :) I used the recommendations for [inline svgs](https://css-tricks.com/accessible-svgs/#aa-2-inline-svg) in 4f6ab7c for a title, description, and aria tags to make sure the description gets picked up. Lmk if this seems appropriate!